templates/index.html

```diff
@@ -1,18 +1,44 @@
-{% extends "layout.html" %}
-{% block title %}To-Do App{% endblock %}
-
-{% block content %}
-  <div class="jumbotron">
-    <h1 class="display-4">To-Do App</h1>
-    <p class="lead">Just another to-do app.</p>
-  </div>
-
-  <div class="row">
-    <div class="col-md-12">
-      <h2>Items</h2>
-      <ul class="list-group mb-4">
-        <!-- List the todo items here -->
-      </ul>
-    </div>
-  </div>
-{% endblock %}
\ No newline at end of file
+<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 4.01//EN" "http://www.w3.org/TR/html4/strict.dtd">
+<html>
+<head>
+  <meta http-equiv="Content-Type" content="text/html; charset=utf-8">
+  <meta http-equiv="Content-Style-Type" content="text/css">
+  <title></title>
+  <meta name="Generator" content="Cocoa HTML Writer">
+  <meta name="CocoaVersion" content="1671.6">
+  <style type="text/css">
+    p.p1 {margin: 0.0px 0.0px 0.0px 0.0px; line-height: 14.0px; font: 12.0px Times; color: #000000; -webkit-text-stroke: #000000}
+    p.p3 {margin: 0.0px 0.0px 12.0px 0.0px; line-height: 14.0px; font: 12.0px Times; color: #000000; -webkit-text-stroke: #000000}
+    p.p5 {margin: 0.0px 0.0px 0.0px 0.0px; line-height: 14.0px; font: 12.0px Times; color: #000000}
+    p.p6 {margin: 0.0px 0.0px 0.0px 0.0px; line-height: 14.0px; font: 10.8px Menlo; color: #2f3339; -webkit-text-stroke: #2f3339; min-height: 13.0px}
+    p.p7 {margin: 0.0px 0.0px 0.0px 0.0px; line-height: 14.0px; font: 12.0px Times; color: #000000; -webkit-text-stroke: #000000; min-height: 12.0px}
+    span.s1 {font-kerning: none}
+    span.s2 {font: 18.0px Times; font-kerning: none}
+  </style>
+</head>
+<body>
+<p class="p1"><span class="s1">{% extends "layout.html" %} {% block title %}To-Do App{% endblock %} {% block content %}</span></p>
+<h1 style="margin: 0.0px 0.0px 16.1px 0.0px; line-height: 28.0px; font: 24.0px Times; color: #000000; -webkit-text-stroke: #000000"><span class="s1"><b>To-Do App</b></span></h1>
+<p class="p3"><span class="s1">Just another to-do app.</span></p>
+<h2 style="margin: 0.0px 0.0px 14.9px 0.0px; line-height: 22.0px; font: 12.0px Times; color: #000000; -webkit-text-stroke: #000000"><span class="s2"><b>Items</b></span><span class="s1"><br>
+{% for listitem in mylist %}</span></h2>
+<p class="p1"><span class="s1">{{ listitem.id }} : {{ listitem.title }} : {{ listitem.status }}</span></p>
+<p class="p5"><span class="s1">{% endfor %}</span></p>
+<form action="/add" method="post">
+  <label>Enter title of new item to be added</label>
+  <br/>
+  <input name="input_title" type="text" placeholder="title" id="exampleInput1">
+  <button type="submit">Add</button>
+</form>
+<br/>
+<form action="/mark" method="post">
+  <label>Enter id of item to be completed</label>
+  <br/>
+  <input name="item_id" type="text" placeholder="id" id="exampleInput2">
```

Comment:
You might to ensure that the application doesn't error out if, say, someone enters `-1` here.